readthedocs/doc_builder/environments.py

```diff
@@ -315,8 +315,8 @@ def run(self):
                 cmd_stdout, cmd_stderr = out
             else:
                 cmd_stdout = out
-            self.output = self.sanitize_output(cmd_stdout)
-            self.error = self.sanitize_output(cmd_stderr)
+            self.output = self.sanitize_output(cmd_stdout or "")
+            self.error = self.sanitize_output(cmd_stderr or "")
```

Comment:
This feels a bit magic. Can we do this on a separate line w/ a comment, so it's more obvious what we're doing?